### PR TITLE
Autocomplete variant name

### PR DESCRIPTION
**Pull request by Zales0123**
zrzut ekranu 2018-06-12 o 09 07 25

### CODE REVIEW

File: src/Resources/config/app/routing.yml [hunk 1]
```diff
@@ -46,5 +46,31 @@ sylius_admin_order_creation_ajax_customer_by_email:
                 method: findBy
                 arguments: [email: $email]
 
+sylius_admin_order_creation_ajax_product_variants_by_phrase:
```
Comment (GSadee) [inline src/Resources/config/app/routing.yml]: Maybe this is time to extract that to a separate file (`routing/ajax.yml`)

File: tests/Behat/Resources/services.xml [hunk 2]
```diff
@@ -2,12 +2,17 @@
...
+            <argument type="service" id="Tests\Sylius\AdminOrderCreationPlugin\Behat\Service\AutoCompleteSelector" />
+        </service>
         <service id="Tests\Sylius\AdminOrderCreationPlugin\Behat\Page\Admin\OrderCreatePage" parent="sylius.behat.page.admin.crud.create">
             <argument>sylius_admin_order_creation_create_order</argument>
+            <argument type="service" id="Tests\Sylius\AdminOrderCreationPlugin\Behat\Service\AutoCompleteSelector" />
         </service>
         <service id="Tests\Sylius\AdminOrderCreationPlugin\Behat\Page\Admin\OrderIndexPage" parent="sylius.behat.page.admin.order.index" />
-        <service id="Tests\Sylius\AdminOrderCreationPlugin\Behat\Page\Admin\OrderShowPage" parent="sylius.behat.page.admin.order.show" />
+        <service id="Tests\Sylius\AdminOrderCreationPlugin\Behat\Page\Admin\OrderShowPage" parent="sylius.behat.page.admin.order.show">
+
```
Comment (GSadee) [inline tests/Behat/Resources/services.xml]: ?

Comment (Zales0123) [reply tests/Behat/Resources/services.xml]: ????? :O :O :O :O :O 😱

**Comment by Zales0123** [comment]
@GSadee fixed

**Comment by GSadee** [comment]
Good job Mati! 🍯

**Pull request merged**